pkg/controller/controller_broker_test.go

```diff
@@ -133,12 +138,33 @@ func TestShouldReconcileServiceBroker(t *testing.T) {
 				broker := getTestServiceBrokerWithStatus(v1alpha1.ConditionTrue)
 				broker.Generation = 2
 				broker.Status.ReconciledGeneration = 1
+				broker.Spec.RelistDuration = &metav1.Duration{3 * time.Hour}
 				return broker
 			}(),
 			now:       time.Now(),
-			interval:  3 * time.Hour,
 			reconcile: true,
 		},
+		{
+			name: "ready, duration behavior, nil duration",
```

Comment:
We shouldn't need to test this. This should be caught by the Broker validator.

Reply:
If it's being asserted in `shouldReconcileBroker`,  I think it's worth keeping as a test case. Happy to remove if folks feel strongly otherwise.

Reply:
Keep it